index.html

```diff
@@ -46,8 +46,10 @@ <h3>haiku</h3>
        </main>
     <footer>
         <hr>
-        <blockquote>God is great, beer is good, and people are crazy </blockquote>
-        <cite>&mdash;Author <span class="author">Unknown.</span></cite>
+        <blockquote>God is great, beer is good, and people are crazy 
+            <cite>&mdash;Author <span class="author">Unknown.</span></cite>
+        </blockquote>
+
```

Comment:
Lab instructions were to put `<cite>` inside `<blockquote>`

Reply:
Oh sorry I was helped with that, ill fix that.